apps/nextjs-website/src/components/rendering/rendering.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { HeroProps } from '@pagopa/pagopa-editorial-components/dist/components/Hero';
+import Hero from '../Hero';
+import { SectionDataToHeroProps } from './sectionDataToProps';
+import { PageSection } from '@/lib/fetch/types/PageSection';
+
+export function rendering(componentData: PageSection) {
+  // Temporarily disable no-small-switch rule (new cases will be added with PRs for other components)
+  // eslint-disable-next-line no-underscore-dangle,sonarjs/no-small-switch
+  switch (componentData.__component) {
+    case 'sections.hero':
+      const HeroSectionProps: HeroProps = SectionDataToHeroProps(componentData);
```

Comment:
I would remove the return type, letting the compiler infer it.

If you do that you can also remove the [related import](https://github.com/pagopa/b2b-portals/pull/83/files#diff-bf308668e57f244359b43261e1e4eb72b66f520fe64e7100dbca2f2489dc35f8R2).

```suggestion
      const HeroSectionProps = SectionDataToHeroProps(componentData);
```